hail_scripts/v01/utils/computed_fields.py

```diff
@@ -140,7 +140,7 @@ def get_expr_for_vep_sorted_transcript_consequences_array(vep_root="va.vep", inc
                         "other"
             })
         ).sortBy(c => let
-            is_coding = (orElse(c.biotype, "") == "protein_coding") and
+            is_coding = (orElse(c.biotype, "") == "protein_coding" && c.major_consequence_rank <= CONSEQUENCE_TERM_RANK_LOOKUP.get("missense_variant").toInt) and
```

Comment:
The code looks fine to me, except would replace `missense_variant` => `coding_sequence_variant`